src/flows/android.js

```diff
@@ -4,31 +4,49 @@ module.exports.runAndroid = async (toolbox, config) => {
   await setupGradle(toolbox, config)
 }
 
-const askQuestion = async (prompt, options) => {
+const askQuestions = async (prompt, options) => {
   const askGooglePlayJSONPath = {
     type: 'input',
     initial: options.googleJsonPath,
     skip: () => options.googleJsonPath,
     name: 'googleJsonPath',
     message: 'Path to Google Play Store JSON?'
   }
-  const { googleJsonPath } = await prompt.ask(askGooglePlayJSONPath)
-  return {
-    googleJsonPath: options.googleJsonPath,
-    googleJsonPath
+  const askStorePassword = {
```

Comment:
It would be great to first make sure that user wants to create a keystore and only then asks these.

Reply:
I just ran into this as my existing configuration got mangled; for existing apps, prompting for existing keystore configurations would be great.